docs/concepts/mvcc-read-conflicts.md

```diff
@@ -0,0 +1,100 @@
+# MVCC_READ_CONFLICT Errors
+
+## Overview
+
+MVCC_READ_CONFLICT errors in Hyperledger Fabric occur during transaction validation when multiple transactions attempt to modify the same state concurrently. These errors are a consequence of Fabric's Multi-Version Concurrency Control (MVCC) system, which ensures that changes applied to the ledger remain consistent across the network.
```

Comment:
worth to mention that there is also `PHANTOM_READ_CONFLICT`, which is basically the same, but for range reads